fastlane_bot/bot.py

```diff
@@ -636,6 +636,8 @@ def validate_pool_data(self, arb_opp):
                 "tkn1_symbol": current_pool.tkn1_symbol,
                 "tkn0_decimals" : current_pool.tkn0_decimals,
                 "tkn1_decimals": current_pool.tkn1_decimals,
+                "descr": current_pool.descr,
+                "last_updated_block": current_pool.last_updated_block
```

Comment:
Please point out which tests this change fixes (i.e., tests which fail without it and pass with it).
If no such tests exist, then please add one.